vendor/k8s.io/kubernetes/pkg/kubectl/cmd/testdata/edit/testcase-apply-edit-last-applied-syntax-error/2.original

```diff
@@ -1,7 +1,7 @@
 # Please edit the 'last-applied-configuration' annotations below.
 # Lines beginning with a '#' will be ignored, and an empty file will abort the edit.
 #
-# The edited file had a syntax error: error parsing edited-file: error converting YAML to JSON: yaml: line 12: could not find expected ':'
+# The edited file had a syntax error: error parsing edited-file: error converting YAML to JSON: yaml: line 13: could not find expected ':'
```

Comment:
@deads2k I'm not sure how this worked before or if it did. So 'spec' is missing the ':'. In the 2.original file it would be normally indicated by the editor at line 12 (it does not include the two comment lines the editor adds telling you what the issue is so that's why it's 12 versus 14). However the 2.edited file adds a line, 'new-label1: foo1', which therefore makes the line number 13. So two changes occur, add a line and remove the ':'. These tests are not in 4.x.

Reply:
This is the test case that fails without my change:
vendor/k8s.io/kubernetes/pkg/kubectl/cmd TestEdit apply-edit-last-applied-syntax-error
See full failure message below.
See my full explanation in comment above. In short, I tested this on a 3.11 and 3.10 cluster and both are consistent with my change. The parsing is fine. What's driving the change is the 'editor' function leaves the added line in when it displays to the user so perhaps that functionality has changed at some point, e.g. if it displayed the original file with only the missing colon and not the added line the line number would be correct.

--- FAIL: TestEdit (1.43s)
    --- PASS: TestEdit/apply-edit-last-applied (0.07s)
    --- PASS: TestEdit/apply-edit-last-applied-list (0.06s)
    --- PASS: TestEdit/apply-edit-last-applied-list-fail (0.09s)
	edit_test.go:126: apply-edit-last-applied-syntax-error, step 2: diff in edit content:
		# Please edit the 'last-applied-configuration' annotations below.
		# Lines beginning with a '#' will be ignored, and an empty file will abort the edit.
		#
		# The edited file had a syntax error: error parsing edited-file: error converting YAML to JSON: yaml: line 1
		
		A: 3: could not find expected ':'
		#
		kind: Service
		metadata:
		  annotations: {}
		  labels:
		    app: svc1
		    new-label: foo
		  name: svc1
		  namespace: myproject
		spec
		  ports:
		    name: "80"
		    port: 81
		    protocol: TCP
		    targetPort: 81
		  sessionAffinity: None
		  type: ClusterIP
		status:
		  loadBalancer: {
		
		
		B: 2: could not find expected ':'
		#
		kind: Service
		metadata:
		  annotations: {}
		  labels:
		    app: svc1
		    new-label: foo
		  name: svc1
		  namespace: myproject
		spec
		  ports:
		    name: "80"
		    port: 81
		    protocol: TCP
		    targetPort: 81
		  sessionAffinity: None
		  type: ClusterIP
		status:
		  loadBalancer: {